features/admin.workflow-requests.v1/components/active-filters-bar.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Button, Icon } from "semantic-ui-react";
+import "../styles/workflow-requests.css";
+
+export interface FilterTag {
+    key: string;
+    label: string;
+    type: string;
+    value: string;
+}
+
+interface ActiveFiltersBarProps {
+    filters: FilterTag[];
+    onClearAll: () => void;
+    onRemove: (filter: FilterTag) => void;
+    [ key: string ]: any;
+}
+
+const ActiveFiltersBar: React.FC<ActiveFiltersBarProps> = (
+    { filters, onRemove, onClearAll, ..._rest }: ActiveFiltersBarProps
+) => {
+    const { t } = useTranslation([ "workflow-requests" ]);
```

Comment:
Just this would be enough, wouldn't it?

```suggestion
    const { t } = useTranslation();
```